src/tqdm_publisher/_handler.py

```diff
@@ -31,7 +31,7 @@ def on_progress_update(progress_update: dict):
             """
             self.announce(message=dict(**progress_update, **additional_metadata))
 
-        return TQDMProgressSubscriber(iterable=iterable, on_progress_update=on_progress_update, **tqdm_kwargs)
+        return TQDMProgressSubscriber(*tqdm_args, on_progress_update=on_progress_update, **tqdm_kwargs)
```

Comment:
```suggestion
        return TQDMProgressSubscriber(*tqdm_args, *, on_progress_update=on_progress_update, **tqdm_kwargs)
```

Actually, to match the statement in the CHANGELOG (required), need this star here

Reply:
This does not seem acceptable when instantiating a class. Is this where you meant to put this?

Reply:
No, meant in the init

Reply:
This is already handled by [the `*tqdm_args` argument](https://github.com/catalystneuro/tqdm_publisher/blob/f6e9a8fcb83e42c7d7ac64e64ef7f021bf8bcea0/src/tqdm_publisher/_subscriber.py#L7), right?

Reply:
No, `*` says 'everything to the right of this position is required to be a keyword argument only'

Reply:
`*` by itself that is, `*args` just unpacks a list of ordered unnamed positional values to the signature

Reply:
Actually I guess you're right, if it sees `*args` it imposes keyword only to the right of that